actions/class.Runner.php

```diff
@@ -608,7 +608,6 @@ public function move()
             $this->saveItemResponses(false);
             $this->saveToolStates();
 
-            $serviceContext->getTestSession()->initItemTimer();
```

Comment:
I wonder if the timeline is still correct after this change. The time range has been closed at line [602](https://github.com/oat-sa/extension-tao-testqti/pull/1947/files#diff-fea95e24360d827ee2d0e75d5c4d18bc5d43134b522acb2f205ad18f7be19a67R602). A new time range needs to be started, otherwise, time won't be properly counted, and issues may occur.

Maybe would it be better to apply a similar condition instead:
```php
if (!$this->getRunnerService()->isTerminated($serviceContext)) {
    $serviceContext->getTestSession()->initItemTimer();
}
```